tasks/main.yml

```diff
@@ -5,6 +5,10 @@
     - "{{ role_path }}/vars/{{ ansible_os_family }}.yml"
     - "{{ role_path }}/vars/{{ ansible_os_family }}-php{{ php_version }}.yml"
 
+- name: Remove missing JSON extension for PHP 8.0 (included by default)
+  set_fact: "{{ __php_packages }}={{ __php_packages | reject('search','php8.0-json') | list }}"
+  when: __php_packages is defined and php_version == "8.0"
```

Comment:
Can you make this use the version compare filter? https://docs.ansible.com/ansible/latest/user_guide/playbooks_tests.html#comparing-versions

```
php_version is version('8.0', '>=')
```

And maybe break the when up into two lines for easier scanning:

```yaml
  when:
    - __php_packages is defined
    - php_version is version('8.0', '>=')
```

Reply:
Sure. Do you have a good way to test these roles locally? Is it a matter of setting up molecule or something?

Reply:
Yeah; `pip3 install molecule` then `MOLECULE_DISTRO=ubuntu1804 playbook=8.0.yml molecule test`

Note that the directory for this git checkout needs to be named 'geerlingguy.php-versions' for molecule to read it properly.

Reply:
10-4 thanks.

Reply:
My local test with molecule is stuck on php 7.3. That happens running the other playbooks for 7.2 and 7.4 as well.
```
MOLECULE_DISTRO=ubuntu1804 playbook=7.2.yml molecule test
MOLECULE_DISTRO=ubuntu1804 playbook=7.4.yml molecule test
MOLECULE_DISTRO=ubuntu1804 playbook=8.0.yml molecule test
```

Reply:
Oh shoot, that should actually be `MOLECULE_PLAYBOOK` not `playbook`. Sorry about that!

Reply:
No worries. Thanks for the assist.